src/pixbuf_animation.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013-2020, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use PixbufAnimation;
+use PixbufAnimationIter;
+use glib::object::IsA;
+use glib::translate::*;
+use glib::TimeVal;
+
+pub trait PixbufAnimationExtManual {
+    fn get_iter(&self, start_time: Option<TimeVal>) -> PixbufAnimationIter;
```

Comment:
```suggestion
    fn get_iter(&self, start_time: Option<&TimeVal>) -> PixbufAnimationIter;
```

Probably also for the others. But maybe there's a better Rust type for these things?

In `std` (non-public) there's:

```rust
    impl From<libc::timeval> for SystemTime {
        fn from(t: libc::timeval) -> SystemTime {
            SystemTime::from(libc::timespec {
                tv_sec: t.tv_sec,
                tv_nsec: (t.tv_usec * 1000) as libc::c_long,
            })
        }
    }
```

So maybe `SystemTime` would be a good choice here

Reply:
See also https://github.com/gtk-rs/gio/issues/320